src/main/java/com/sumologic/jenkins/jenkinssumologicplugin/PluginDescriptorImpl.java

```diff
@@ -97,7 +98,7 @@ public String getDisplayName() {
     @Override
     public boolean configure(StaplerRequest req, JSONObject formData) throws FormException {
         boolean configOk = super.configure(req, formData);
-        url = formData.getString("url");
+        url = Secret.fromString(formData.getString("url"));
```

Comment:
I suppose there is some request to make this a secret instead of plain text?

Reply:
https://github.com/SumoLogic/sumologic-jenkins-plugin/pull/36 - request from a user to make it secret.